### PR TITLE
Feature/share threads

### PR DESCRIPTION
**Pull request by Arkirion**
image

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -35,6 +35,41 @@ const config = {
...
+      '1122388627557732362', // javascript
+      '1122390683106414652', // python
+      '1122390294973915176', // java-kotlin
+      '1122391769775079505', // php
+      '1122393447698014271', // c
+      '1122396855775539320', // c++
+      '1122393447698014271', // c-sharp-dotnet
+      '1122400137214427176', // go
+      '1122399598107967580', // rust
+      '1122399272499941426', // dart
```
Comment (Chere3) [inline src/config.ts:51]: Very bad practice, use the parentChannel instead to fetch all the channels in category.
https://old.discordjs.dev/#/docs/discord.js/main/class/ThreadChannel?scrollTo=parent

**Review by Chere3 — CHANGES_REQUESTED** [review]

File: src/config.ts (hunk 1 above)
Comment (Arkirion) [reply src/config.ts:51]: We are not adding all forums

Comment (Arkirion) [reply src/config.ts:51]: also some forums are on different categories

Comment (Arkirion) [reply src/config.ts:51]: we need storage in order to avoid this config =)

Comment (Chere3) [reply src/config.ts:51]: In that case it is better to create two properties, one of the categories to be obtained, and one of the channels to be ignored within those categories

```json
{
   forumTargets: ["category1", "category2"],
   ignoredChannels: ["channel1", "channel2"]
}

Comment (Chere3) [reply src/config.ts:51]: And you can leave a channel property, for channels that are outside the categories, such as the math channel

Comment (Arkirion) [reply src/config.ts:51]: as I said on Discord, I understand your approach, however ( ignoring that we are using a file to set this config ) is more easier just add or remove ids, Im against these conditions that are not that clear like just add or remove things, even if it mean a lot of ids

File: src/config.ts [hunk 2]
```diff
@@ -35,6 +35,41 @@ const config = {
...
+      '1019727139173576814', // hard-ware
+      '1019789271386837102', // linux
+      '805086058382491648', // matemáticas
+      '1019776581599768719', // seguridad-informatica
+      '1019773997296123944', // redes
+      '1019719246655258705', // windows
+    ],
+    outputChannel: {
+      chat_programadores: {
+        id: '807385882868580392'
```
Comment (Chere3) [inline src/config.ts:69]: follow the format of the channels ids.
Property id is unnecesary.

```json
"outputChannel": {
  "chat_programadores": "807385882868580392"
 }

**Review by Chere3 — CHANGES_REQUESTED** [review]

**Review by Chere3 — APPROVED** [review]

**Pull request merged**